libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ApolloRequest.kt

```diff
@@ -59,10 +60,19 @@ private constructor(
 
     override var httpHeaders: List<HttpHeader>? = null
 
+    val additionalHttpHeaders: List<HttpHeader>?
```

Comment:
Yes because `ApolloRequest` is in `apollo-api` and we need to use it [here](https://github.com/apollographql/apollo-kotlin/blob/9f65ed9277702aa84510fca9ed46ab788c4ad1ec/libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/ApolloClient.kt#L139) from `apollo-runtime`

Reply:
But this one is on the Builder

Reply:
Ah my bad. Yea maybe not although all the other ones have it. Don't know. Could be useful to get what's inside a builder?

PS: What we definitely don't need though is setters on all other properties.

Reply:
> Actually I think we should have `private set` on all the `override var`?

Yea exactly. But this is breaking, I didn't want to do it in the same PR in case we want this for 3.7.5. I'll open a follow up PR